### PR TITLE
Fix create-package script

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
## Overview

Fixes `create-package` script.

Changes are:

- Remove jest DOM config step (https://github.com/OfficeDev/office-ui-fabric-react/commit/0379d3c1d1ac82d654170375f9d22b574e17c45c)
- Remove trailing comma causing parse error when reading `package.json` template (https://github.com/OfficeDev/office-ui-fabric-react/commit/0535118f7476901633c2830b661df43e04e6a27b)
    - Will see if read JSON can be made more robust to handle this as part of https://github.com/OfficeDev/office-ui-fabric-react/issues/9494.
- Consolidate `rush install` and `rush update` (https://github.com/OfficeDev/office-ui-fabric-react/commit/8e34c929f0b0e0dc066a0b1bdec4a1c460d33b00)
- Add `@uifabric/build` to `package.json` template (https://github.com/OfficeDev/office-ui-fabric-react/commit/b002236422128ebdcab0895f0881300a4154da5c, https://github.com/OfficeDev/office-ui-fabric-react/commit/8c37aa08fc64e6773ba4980b0c330ef994e43d6e)

## Success output

```shell
~\Work\office-ui-fabric-react [keco/fix-create-package-script ↑4]> npm run create-package foo                           
> office-ui-fabric-react-repo@7.0.0 create-package ~\office-ui-fabric-react
> node scripts/create-package.js "foo"

Creating .npmignore...
Creating .npmrc...
Creating CHANGELOG.json...
Creating CHANGELOG.md...
Creating LICENSE...
Creating README.md...
Creating index.html...
Creating just.config.js...
Creating jest.config.js...
Creating jsconfig.json...
Creating package.json...
Creating prettier.config.js...
Creating tsconfig.json...
Creating tslint.json...
Creating webpack.config.js...
Creating webpack.serve.config.js...
Creating config\tests.js...
Creating config\pre-copy.json...
Creating src\common\tests.js...
Creating src\index.ts...
Creating src\version.ts...
Creating src\demo\AppDefinition.tsx...
Creating src\demo\GettingStartedPage.tsx...
Creating src\demo\index.tsx...
New package foo successfully created.
Running "npm generate" (to bypass this step, use --no-generate arg)

> office-ui-fabric-react-repo@7.0.0 generate ~\office-ui-fabric-react
> npm run rush-update


> office-ui-fabric-react-repo@7.0.0 rush-update ~\office-ui-fabric-react
> node common/scripts/install-run-rush.js update

The rush.json configuration requests Rush version 5.6.0

Invoking "rush update"
----------------------



Rush Multi-Project Build Tool 5.6.0 - https://rushjs.io


Starting "rush update"

Warning: A phantom "node_modules" folder was found. This defeats Rush's protection against NPM phantom dependencies and
may cause confusing build errors. It is recommended to delete this folder:
"~\office-ui-fabric-react\node_modules"


Found files in the "common/git-hooks" folder.
Successfully installed these Git hook scripts: pre-commit

Trying to acquire lock for pnpm-2.19.2
Acquired lock for pnpm-2.19.2
Found pnpm version 2.19.2 in C:\Users\keco\.rush\node-v10.15.0\pnpm-2.19.2

Symlinking "~\office-ui-fabric-react\common\temp\pnpm-local"
  --> "C:\Users\keco\.rush\node-v10.15.0\pnpm-2.19.2"

Updating temp projects in ~\office-ui-fabric-react\common\temp\projects
Copying ~\office-ui-fabric-react\common\config\rush\.npmrc --> ~\office-ui-fabric-react\common\temp\.npmrc
Updating ~\office-ui-fabric-react\common\temp\pnpmfile.js
Updating ~\office-ui-fabric-react\common\temp\projects\foo.tgz
Finished creating temporary modules (0.54 seconds)

The shrinkwrap file is missing the following dependencies:
  "@microsoft/load-themed-styles" (^1.7.13) required by "@uifabric/foo"
  "@types/enzyme" (3.1.13) required by "@uifabric/foo"
  "@types/enzyme-adapter-react-16" (1.0.3) required by "@uifabric/foo"
  "@types/es6-promise" (0.0.32) required by "@uifabric/foo"
  "@types/jest" (23.0.0) required by "@uifabric/foo"
  "@types/react" (16.8.11) required by "@uifabric/foo"
  "@types/react-dom" (16.8.4) required by "@uifabric/foo"
  "@types/react-test-renderer" (^16.0.0) required by "@uifabric/foo"
  "@types/webpack-env" (1.13.9) required by "@uifabric/foo"
  "enzyme" (~3.9.0) required by "@uifabric/foo"
  "enzyme-adapter-react-16" (^1.2.0) required by "@uifabric/foo"
  "es6-weak-map" (^2.0.2) required by "@uifabric/foo"
  "react" (16.8.6) required by "@uifabric/foo"
  "react-dom" (16.8.6) required by "@uifabric/foo"
  "react-test-renderer" (^16.3.0) required by "@uifabric/foo"
  "tslib" (^1.7.1) required by "@uifabric/foo"


Checking node_modules in ~\office-ui-fabric-react\common\temp

Deleting: ~\office-ui-fabric-react\common\temp\rush-link.json

Running "pnpm install" in ~\office-ui-fabric-react\common\temp

Using hooks from: ~\office-ui-fabric-react\common\temp\pnpmfile.js
readPackage hook is declared. Manifests of dependencies might get overridden
 WARN  deprecated github@7.3.2: 'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)
Already up-to-date
Resolving: total 1900, reused 1849, downloaded 41, done
 WARN  Cannot link bin "jest" of "jest-cli" to "~\office-ui-fabric-react\common\temp\node_modules\.bin". A package called "jest" already has its bin linked.
Updating ~\office-ui-fabric-react\common\config\rush\shrinkwrap.yaml

Linking projects together...

LINKING: @uifabric/api-docs
Purging ~\office-ui-fabric-react\packages\api-docs\node_modules

LINKING: @uifabric/azure-themes
Purging ~\office-ui-fabric-react\packages\azure-themes\node_modules

LINKING: @uifabric/build
Purging ~\office-ui-fabric-react\scripts\node_modules

LINKING: @uifabric/charting
Purging ~\office-ui-fabric-react\packages\charting\node_modules

LINKING: @uifabric/codepen-loader
Purging ~\office-ui-fabric-react\packages\codepen-loader\node_modules

LINKING: @uifabric/date-time
Purging ~\office-ui-fabric-react\packages\date-time\node_modules

LINKING: @uifabric/example-app-base
Purging ~\office-ui-fabric-react\packages\example-app-base\node_modules

LINKING: @uifabric/experiments
Purging ~\office-ui-fabric-react\packages\experiments\node_modules

LINKING: @uifabric/fabric-website
Purging ~\office-ui-fabric-react\apps\fabric-website\node_modules

LINKING: @uifabric/fabric-website-resources
Purging ~\office-ui-fabric-react\apps\fabric-website-resources\node_modules

LINKING: @uifabric/file-type-icons
Purging ~\office-ui-fabric-react\packages\file-type-icons\node_modules

LINKING: @uifabric/fluent-theme
Purging ~\office-ui-fabric-react\packages\fluent-theme\node_modules

LINKING: @uifabric/foo
Purging ~\office-ui-fabric-react\packages\foo\node_modules

LINKING: @uifabric/foundation
Purging ~\office-ui-fabric-react\packages\foundation\node_modules

LINKING: @uifabric/foundation-scenarios
Purging ~\office-ui-fabric-react\packages\foundation-scenarios\node_modules

LINKING: @uifabric/icons
Purging ~\office-ui-fabric-react\packages\icons\node_modules

LINKING: @uifabric/jest-serializer-merge-styles
Purging ~\office-ui-fabric-react\packages\jest-serializer-merge-styles\node_modules

LINKING: @uifabric/lists
Purging ~\office-ui-fabric-react\packages\lists\node_modules

LINKING: @uifabric/merge-styles
Purging ~\office-ui-fabric-react\packages\merge-styles\node_modules

LINKING: @uifabric/migration
Purging ~\office-ui-fabric-react\packages\migration\node_modules

LINKING: @uifabric/pr-deploy-site
Purging ~\office-ui-fabric-react\apps\pr-deploy-site\node_modules

LINKING: @uifabric/prettier-rules
Purging ~\office-ui-fabric-react\packages\prettier-rules\node_modules

LINKING: @uifabric/react-cards
Purging ~\office-ui-fabric-react\packages\react-cards\node_modules

LINKING: @uifabric/set-version
Purging ~\office-ui-fabric-react\packages\set-version\node_modules

LINKING: @uifabric/styling
Purging ~\office-ui-fabric-react\packages\styling\node_modules

LINKING: @uifabric/test-utilities
Purging ~\office-ui-fabric-react\packages\test-utilities\node_modules

LINKING: @uifabric/theme-samples
Purging ~\office-ui-fabric-react\packages\theme-samples\node_modules

LINKING: @uifabric/tslint-rules
Purging ~\office-ui-fabric-react\packages\tslint-rules\node_modules

LINKING: @uifabric/utilities
Purging ~\office-ui-fabric-react\packages\utilities\node_modules

LINKING: @uifabric/variants
Purging ~\office-ui-fabric-react\packages\variants\node_modules

LINKING: @uifabric/webpack-utils
Purging ~\office-ui-fabric-react\packages\webpack-utils\node_modules

LINKING: a11y-tests
Purging ~\office-ui-fabric-react\apps\a11y-tests\node_modules
createFolderWithRetry() stalled for 3.77 seconds

LINKING: dom-tests
Purging ~\office-ui-fabric-react\apps\dom-tests\node_modules

LINKING: office-ui-fabric-react
Purging ~\office-ui-fabric-react\packages\office-ui-fabric-react\node_modules

LINKING: perf-test
Purging ~\office-ui-fabric-react\apps\perf-test\node_modules

LINKING: server-rendered-app
Purging ~\office-ui-fabric-react\apps\server-rendered-app\node_modules

LINKING: ssr-tests
Purging ~\office-ui-fabric-react\apps\ssr-tests\node_modules

LINKING: test-bundles
Purging ~\office-ui-fabric-react\apps\test-bundles\node_modules

LINKING: theming-designer
Purging ~\office-ui-fabric-react\apps\theming-designer\node_modules

LINKING: todo-app
Purging ~\office-ui-fabric-react\apps\todo-app\node_modules

LINKING: vr-tests
Purging ~\office-ui-fabric-react\apps\vr-tests\node_modules
Writing "~\office-ui-fabric-react\common\temp\rush-link.json"

Linking finished successfully. (9.75 seconds)

Next you should probably run "rush build" or "rush rebuild"

Rush update finished successfully. (1 minute 54.7 seconds)
```

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9491)

### CODE REVIEW

File: scripts/create-package.js [hunk 1]
```diff
@@ -52,30 +52,102 @@ const experimentsPackageJson = JSON.parse(fs.readFileSync(experimentsPackagePath
 
 // Steps (mustache template names and output file paths)
 const steps = [
-  { template: 'NpmIgnore', output: '.npmignore' },
```
Comment (KevinTCoughlin) [inline scripts/create-package.js]: My prettier extension may have caused this. Will clean it up in the AM 😴.

**Comment by cliffkoh** [comment]
@msft-github-bot please make sure @ecraig12345 or @kenotron approves.

File: scripts/create-package.js (hunk 1 above)
Comment (ecraig12345) [reply scripts/create-package.js]: I wouldn't expect Prettier to make this particular change--maybe it was the default VS Code JS formatter or some other formatting extension?

File: scripts/templates/create-package/EmptyPackageJson.mustache [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "start": "npx just-scripts dev",
     "start-test": "npx just-scripts jest-watch",
     "update-snapshots": "npx just-scripts jest -u",
-    "just": "npx just-scripts",
+    "just": "npx just-scripts"
```
Comment (ecraig12345) [inline scripts/templates/create-package/EmptyPackageJson.mustache]: Thanks! Re: your comment in the PR description about making our JSON reading more robust, if you wanted you could use `jju` (already a dep of the scripts package), which appears to handle trailing commas. What would be even better is just adding a test for the create-package script, but that's certainly not something you need to do right now.

Comment (KevinTCoughlin) [reply scripts/templates/create-package/EmptyPackageJson.mustache]: > What would be even better is just adding a test for the create-package script, but that's certainly not something you need to do right now.

Agreed. Creating an issue for myself as I'll be on shield starting next week -- https://github.com/OfficeDev/office-ui-fabric-react/issues/9494.

File: scripts/create-package.js (hunk 1 above)
Comment (KevinTCoughlin) [reply scripts/create-package.js]: Resolved in latest 3 commits

**Pull request merged**